src/System.IO.Compression/tests/DeflateStreamTests.cs

```diff
@@ -0,0 +1,379 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using System.Reflection;
+using System.IO.Compression;
+using Xunit;
+
+namespace System.IO.Compression.Tests
+{
+    public partial class DeflateStreamTests
+    {
+        static string gzTestFile(String fileName) { return Path.Combine("GZTestData", fileName); }
+
+        [Fact]
+        public static void BaseStream1()
+        {
+            var writeStream = new MemoryStream();
+            var zip = new DeflateStream(writeStream, CompressionMode.Compress);
+
+            Assert.True(zip.BaseStream == writeStream, "Reference not equal with base stream");
```

Comment:
In some cases that makes sense, e.g. if the message can include iteration or other input information.  In this case, though, does the message tell us anything not already available from the assert itself?